templates/index

```diff
@@ -132,10 +132,12 @@ cisco_asa_show_version.textfsm, .*, cisco_asa, sh[[ow]] ver[[sion]]
 cisco_asa_show_route.textfsm, .*, cisco_asa, sh[[ow]] ro[[ute]]
 cisco_asa_show_xlate.textfsm, .*, cisco_asa, sh[[ow]] x[[late]]
 cisco_asa_show_name.textfsm, .*, cisco_asa, sh[[ow]] nam[[e]]
-cisco_asa_show_arp.textfsm, .*, cisco_asa, sh[[ow]] arp
+cisco_asa_show_arp.textfsm, .*, cisco_(asa|ftd), sh[[ow]] arp
 cisco_asa_show_nat.textfsm, .*, cisco_asa, sh[[ow]] nat
 cisco_asa_dir.textfsm, .*, cisco_asa, dir
 
+cisco_ftd_ping.textfsm, .*, cisco_(asa|ftd), ping
```

Comment:
Sorry about this! I should have been clearer. Do you mind moving this up and then renaming to `cisco_asa_ping.textfsm`? I think we'll keep the naming convention the same, but introduce the `cisco_(asa|ftd)` into the index. The tests will have to be updated as well.

Let me know if you have any issues with this in our public Slack.

Reply:
OK, should be fixed now.